src/main/java/task/Task.java

```diff
@@ -0,0 +1,60 @@
+package task;
```

Comment:
Since any task must be either a todo, a deadline or an event, maybe you can consider making this an abstract class?